src/main/resources/index.html

```diff
@@ -167,6 +167,12 @@
         }
         document.addEventListener('onresize', update);
 
+        function conditionalUpdate() {
+            if (isClickAction) {
+                update();
+            }
+        }
```

Comment:
Since this `conditionalUpdate` function is only used in one place, consider inlining the function definition.